src/BinaryParsers/MachOBinary/SingleMachOBinary.cs

```diff
@@ -16,11 +16,33 @@ public class SingleMachOBinary : BinaryBase, IDwarfBinary
         private const string SECTIONNAME_TEXT = "__text";
         private const string SECTIONNAME_DATA = "__data";
         private const string SECTIONNAME_EH_FRAME = "__eh_frame";
+        private const string SECTIONNAME_DEBUG_LOC = "__debug_loc";
```

Comment:
>__debug_loc [](http://example.com/codeflow?start=54&length=11)

I've aggressively added a number of section names we don't currently parse in any way, simply so that others don't need to look for them in the future. These sections names don't match ELF section names precisely and docs seems incrementally harder to locate.